proxy.go

```diff
@@ -10,15 +10,27 @@ package main
 import (
 	"errors"
 	"flag"
+	"fmt"
 	"io"
 	"net"
 	"net/url"
+	"os"
 	"sync"
 
 	"github.com/hashicorp/yamux"
 	"github.com/sirupsen/logrus"
 )
 
+const proxyName = "kata-proxy"
+
+// version is the proxy version. This variable is populated at build time.
+var version = "unknown"
+
+var proxyLog = logrus.WithFields(logrus.Fields{
+	"name": proxyName,
+	"pid":  os.Getpid(),
```

Comment:
I prefer a sandbox name is also added in the log.
But this can be done in later PRs, because a new option is required for this in the command line.

Reply:
Agreed, that would make sense for a follow-on PR.